crates/workspace/src/workspace.rs

```diff
@@ -3503,7 +3489,6 @@ impl Workspace {
                 .map(|(path, _)| path)
                 .collect::<Vec<_>>();
             cx.update(|cx| {
-                cx.clear_recent_documents();
                 cx.add_recent_documents(&current_paths);
```

Comment:
We do not need this addition and, as clippy suggests, the entire `refresh_recent_documents` method.